eva/io_descriptors/eva_arguments.py

```diff
@@ -0,0 +1,140 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from abc import abstractmethod
+from typing import Tuple
+
+from eva.catalog.catalog_type import ColumnType, NdArrayType
+
+
+class IOArgument(object):
+    """
+    Base class for the data types that are used inside Eva. This class is inherited by the NumPyArray,
```

Comment:
The docstring is misleading IMO. This class is purely for interfacing with UDF Input/Outputs. For example Pandas Dataframe option would extend `IOArgument` but is not necessarily a data type that we use inside EVA.